CONTRIBUTING.md

```diff
@@ -20,52 +20,73 @@ slack](https://bit.ly/join-vega-slack).
 
 ### Setting Up Your Environment
 
-Install the latest version of Altair locally using 
-```
-$ pip install git+https://github.com/altair-viz/altair/
-```
-Next step is to fork the repository on GitHub and clone the fork to you local
+Fork the Altair repository on GitHub and clone the fork to you local
 machine. For more details on forking see the [GitHub
 Documentation](https://help.github.com/en/articles/fork-a-repo).
 ```
 $ git clone https://github.com/YOUR-USERNAME/altair.git
 ```
-You can have a single clone of the repository that points to both your fork and
-the main package repository. These pointers to GitHub are called "remotes" .
-On your local clone you should run:
+
+To keep your fork up to date with changes in the this repo,
+you can [use the fetch upstream button on GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).
+
+Now you can install the latest version of Altair locally using `pip`.
+The `-e` flag indicates that your local changes will be reflected
+every time you open a new Python interpreter
+(instead of having to reinstall the package each time).
+```
+$ cd altair/ 
+$ python -m pip install -e .
+```
+
+You can now install the development requirements using
 ```
-$ git remote add upstream https://github.com/altair-viz/altair.git
-$ git checkout master
-$ git pull upstream master
+$ python -m pip install -r requirement_dev.txt
 ```
-And then you'll have all the updates in the master branch of your local fork.
-Note that git will complain if you've committed changes to your local master
-branch that are not on upstream (this is one reason it's good practice to **never**
-work directly on your master branch).
+
 
 ### Creating a Branch
 
-Once your local environment is up-to-date, you can create a new git branch which will
-contain your contribution:
+Once your local environment is up-to-date, you can create a new git branch
+which will contain your contribution
+(always create a new branch instead of making changes to the master branch):
 ```
-$ git checkout -b <branch-name>
+$ git switch -c <branch-name>
 ```
 With this branch checked-out, make the desired changes to the package.
+
 Note that Altair code uses the [black](https://black.readthedocs.io/)
-code formatter, which you can apply to your modifications by installing
-and running black on the local directory:
+and [flake8](https://flake8.pycqa.org/en/latest/) code formatters
```

Comment:
nitpick: flake8 is a linter, not a code formatter

Reply:
Thank you, updated!